parm/fv3lam_rrfs.xml

```diff
@@ -1619,6 +1660,13 @@
         <scale>5.0</scale>
        </param>
 
+       <param>
+       <shortname>AVE_LRGHR_ON_ENTIRE_ATMOS</shortname>
+       <pname>LRGHR</pname>
```

Comment:
@AndrewBenjamin-NOAA I don't see this field generated in your test. You may remove it from this PR, debug and submit a new PR later.